docs/operating_systems/rhel.md

```diff
@@ -25,10 +25,16 @@ rh_subscription_role: "Red Hat Enterprise Server"
 rh_subscription_sla: "Self-Support"
 ```
 
-If the RHEL 7/8 hosts are already registered to a valid Red Hat support subscription via an alternative configuration management approach prior to the deployment of Kubespray, the successful RHEL `subscription-manager` status check will simply result in the RHEL subscription registration tasks being skipped.
+If the RHEL 8/9 hosts are already registered to a valid Red Hat support subscription via an alternative configuration management approach prior to the deployment of Kubespray, the successful RHEL `subscription-manager` status check will simply result in the RHEL subscription registration tasks being skipped.
 
 ## RHEL 8
 
-If you have containers that are using iptables in the host network namespace (`hostNetwork=true`),
-you need to ensure they are using iptables-nft.
+In the RHEL (including the AlmaLinux, RockyLinux, CentOS) 8, if you have containers that are using iptables in the host network namespace (`hostNetwork=true`), you need to ensure they are using iptables-nft.
 An example how k8s do the autodetection can be found [in this PR](https://github.com/kubernetes/kubernetes/pull/82966)
+
+Kernel version 4.18 is lower than the kubeadm 1.32 system verification requirement. Refer to [this PR](https://github.com/kubernetes/system-validators/pull/48). Therefore, you need to add the following configuration to ignore the kubeadm preflight errors:
+
+```ini
+kubeadm_ignore_preflight_errors:
+  - SystemVerification
+```
```

Comment:
I would put that in a more general place and link to it, since other OS will have the same things (Opensuse / old debian / etc)

Reply:
> I would put that in a more general place and link to it, since other OS will have the same things (Opensuse / old debian / etc)

Thanks @VannTen 

It has been added as a `docs/operations/kernel-requirements.md`, to describe the details of kernel, like the https://kubernetes.io/docs/reference/node/kernel-version-requirements/ .

https://github.com/kubernetes-sigs/kubespray/pull/11885/files#diff-7bc83f52c850bee9985a56a165a499130bd4a1a81e3a19622a8fbecedbb8e142R1